docs/lib/utils.tsx

```diff
@@ -0,0 +1,17 @@
+interface NestedStringObject {
+  [key: string]: NestedStringObject | string;
+}
+
+export const iterateTokens = (colors: NestedStringObject, prefix = '') => {
```

Comment:
Can we have some docs here. This is only for the command menu I imagine?

Reply:
No I moved it out from the design-tokens.tsx because I used it also for the command :)

Reply:
Ah just saw it. But what does it do 👀

Reply:
used to iterate through the colors and combine it in the right way e.g. 'bg-surface-sunken' I add it as comment